buildenv/jenkins/jobs/builds/Build-aix_ppc-64

```diff
@@ -33,15 +33,15 @@ pipeline {
                 timestamps {
                     echo 'Output CCACHE stats before running and clear them'
                     sh '''ccache -s -z'''
-                    
+
                     dir('build') {
                         echo 'Configure...'
                         sh '''cmake -Wdev -DCMAKE_XL_CreateExportList="/usr/vac/bin/CreateExportList -X64" -DOMR_DDR=OFF -C../cmake/caches/Travis.cmake ..'''
-                       
+
                         echo 'Compile...'
-                        sh '''make -j8'''
+                        sh '''export CCACHE_EXTRAFILES="$PWD/omrcfg.h" && make -j8'''
```

Comment:
Neat solution. Thanks for this!

Reply:
I'm assuming we need to put this here, rather than the environment block, because we don't actually know until runtime the path of the build directory

Reply:
I think you are correct. It gives most freedom and prevents hardcoding things.